content/en/administrators_guide/run.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Maintaining and running your Datadog installation
+description: Setup and install Datadog as an administrator to avoid pit-falls down the road
+further_reading:
+- link: "getting_started/dashboards/"
+  tag: "Documentation"
+  text: "Getting Started with Dashboards"
+---
+
+In the previous two sections, you understood Datadog through the lens of the plan and build phases, next you'll learn about the run phase. There are a series of tasks, both internal and external, that you need to manage to keep the Datadog service running efficiently. 
+
+## Service tasks
+
+By releasing your new individual Datadog services in a sequential pattern, gaining confidence as the service develops, you can reduce the risks of the installation, and increase adoption from your users. This section lists a sequence of service item releases that optimizes your Datadog service user's experience. The diversity of IT architecture at this level is extensive, so the following guidance does not go into intricate detail. Here are a few highlights:
```

Comment:
I actually missed some items in this section, we don't want to use "Datadog service" here because it's confusing with Service Catalog and a "service" as a "service" , I'll accept your initial changes then go back and change the wording slightly.